src/FakeItEasy.Analyzer/DiagnosticDefinitions.cs

```diff
@@ -10,7 +10,11 @@ internal static class DiagnosticDefinitions
     {
         public static DiagnosticDescriptor UnusedCallSpecification { get; } = CreateDiagnosticDescriptor(nameof(UnusedCallSpecification), "FakeItEasy0001", "FakeItEasy.Usage", DiagnosticSeverity.Error, true);

```

Comment:
Can't split this line, as described by @thomaslevesque  at https://github.com/FakeItEasy/FakeItEasy/pull/678#issuecomment-214536900